jdaviz/configs/mosviz/helper.py

```diff
@@ -5,7 +5,70 @@
 
 class MosViz(ConfigHelper):
     """MosViz Helper class"""
-    _default_configuration = 'mosviz'
+
+    _default_configuration = "mosviz"
+
+    def load_data(
+        self,
+        onedspectra,
+        twodspectra,
+        images,
+        onedspectra_label=None,
+        twodspectra_label=None,
+        images_label=None,
+    ):
```

Comment:
```suggestion
    def load_data(self, onedspectra, twodspectra, images, onedspectra_label=None, 
                  twodspectra_label=None, images_label=None):
```

To be consistent with the rest of the codebase, line breaks should really only happen when necessary and where appropriate.